### PR TITLE
draft CC Statements page

### PR DESCRIPTION
**Pull request by taroface**
Added a page for the embedded Statements page in CC Console.

- This mostly duplicates the [existing page for CRDB](https://www.cockroachlabs.com/docs/v20.2/ui-statements-page.html), but removes mention of the Network Latency page and All Statement Diagnostics link (which touches on the Advanced Debug page), both not available in CC.
- The images reuse the CRDB images. In case the UI is going to look very different, I may need to take new screenshots. However, this is pending resolution of [QA issues](https://cockroachlabs.atlassian.net/browse/CRDB-2375).
- Reorganized side nav to place Monitoring Page and Statements Page underneath a top-level "Monitor Your Cluster" menu item. Sessions and other UI pages should also appear under this menu.

Some questions for reviewers:
- Are all the features documented here intact on CC?
- Is this too much detail for a CC user?

### CODE REVIEW

**Review by thtruo — APPROVED** [review]
Nice, thanks Ryan! I took a look through the preview. The content you included seems to hit the right balance of features supported and does include things that are only available for CC and not CRDB. 
I don't think there are necessarily extraneous details here - what has been included will be useful for any CC user attempting to utilize the Statements page experience. It LGTM!

> Reviewed 6 of 6 files at r1.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/9917#-:-MUuHApe49JF9RzqQSrQ:btvne5l)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @koorosh)

**Comment by mikeCRL** [comment]
@taroface In addition to the review comments above: 

- I tested all the links that point to the `/stable` CRDB docs and they all work, but apparently not in the TeamCity environment. Do you know a way to work around that in order to be able to merge? Or should we always be linking to the docs version that the current CC is running on, and we can do a find/replace in the future?
- Also just confirming its intentional that all the images are thus far 0 bytes. (I assume they are placeholder files.)

**Review by taroface — COMMENTED** [review]
TFTR! I think @lnhsingh mentioend having problems with building the `/stable` links. @yingzhuchin do you have any ideas why this might be erroring out in the build, and how we might resolve?

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/9917#-:-MXIX_p-61WEwcSXmV_1:b2vwxb5)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @koorosh and @mikeCRL)

___
*[cockroachcloud/statements-page.md, line 3 at r1](https://reviewable.io/reviews/cockroachdb/docs/9917#-MUpLMzmIZ-VN6YhntKR-r1-3:-MXIVS9B0kFegsz5tlXu:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/143258e8c98dfb00e5af400203fc5a365344f8d3/cockroachcloud/statements-page.md#L3)):*
<details><summary><i>Previously, mikeCRL (Mike Lewis) wrote…</i></summary><blockquote>

I'm assuming you plan to update the summary before merging. Just wanted to flag it.
</blockquote></details>

Done.
___
*[cockroachcloud/statements-page.md, line 13 at r1](https://reviewable.io/reviews/cockroachdb/docs/9917#-MUpLMzmIZ-VN6YhntKR-r1-13:-MXIVXOv96DXXooJAJUZ:bfetiyq) ([raw file](https://github.com/cockroachdb/docs/blob/143258e8c98dfb00e5af400203fc5a365344f8d3/cockroachcloud/statements-page.md#L13)):*
<details><summary><i>Previously, mikeCRL (Mike Lewis) wrote…</i></summary><blockquote>

There may be style considerations at some point on _view_ vs. _open_, use of _left-hand_, and _navigation_ as a noun. This is fine for now as it matches the original Statements doc and I can't find any conflicting nav references in the other CC docs from a quick scan.
</blockquote></details>

Good points on style. I think we should discuss in the future.
___
*[cockroachcloud/statements-page.md, line 17 at r1](https://reviewable.io/reviews/cockroachdb/docs/9917#-MUpLMzmIZ-VN6YhntKR-r1-17:-MXIVljvAkm4XjOCJrBw:btbjfsc) ([raw file](https://github.com/cockroachdb/docs/blob/143258e8c98dfb00e5af400203fc5a365344f8d3/cockroachcloud/statements-page.md#L17)):*
<details><summary><i>Previously, mikeCRL (Mike Lewis) wrote…</i></summary><blockquote>

```suggestion
By default, this page shows SQL statements from all applications running on the cluster and hides internal CockroachCloud queries.
```
</blockquote></details>

I'd prefer to keep the comma as I find it more readable (I know it makes the flow slightly awkward). Unless that would be grammatically incorrect?
___
*[cockroachcloud/statements-page.md, line 30 at r1](https://reviewable.io/reviews/cockroachdb/docs/9917#-MUpLMzmIZ-VN6YhntKR-r1-30:-MXIW4jo7IGryj4vJju8:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/143258e8c98dfb00e5af400203fc5a365344f8d3/cockroachcloud/statements-page.md#L30)):*
<details><summary><i>Previously, mikeCRL (Mike Lewis) wrote…</i></summary><blockquote>

> If you haven't yet executed any queries in the cluster as a user, this page will initially be blank.

I know this is text from the original page, but:

This makes it seem that I, the CockroachCloud admin who may be reading this, must have executed queries in the cluster in order to have data appear here. However, my assumption is that any user queries will result in data here. If that's right, maybe rephrase to "This page is initially blank on clusters where no queries have yet been executed." 

Also, is it more clear/accurate to say that the table is initially blank, rather than the page?
</blockquote></details>

Done.
___
*[cockroachcloud/statements-page.md, line 37 at r1](https://reviewable.io/reviews/cockroachdb/docs/9917#-MUpLMzmIZ-VN6YhntKR-r1-37:-MXIXGGfD91LyeAddQT_:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/143258e8c98dfb00e5af400203fc5a365344f8d3/cockroachcloud/statements-page.md#L37)):*
<details><summary><i>Previously, mikeCRL (Mike Lewis) wrote…</i></summary><blockquote>

```suggestion
Statement | SQL statement [fingerprint](#sql-statement-fingerprints).<br><br>To view additional details, click the SQL statement fingerprint to open its [**Statement Details** page](#statement-details-page).
```
</blockquote></details>

Done.
___
*[cockroachcloud/statements-page.md, line 40 at r1](https://reviewable.io/reviews/cockroachdb/docs/9917#-MUpLMzmIZ-VN6YhntKR-r1-40:-MXIXMW_9aYpLglD5SB4:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/143258e8c98dfb00e5af400203fc5a365344f8d3/cockroachcloud/statements-page.md#L40)):*
<details><summary><i>Previously, mikeCRL (Mike Lewis) wrote…</i></summary><blockquote>

Extra space before `<br><br>` here and on subsequent rows, just FYI, though I assume that doesn't matter.
</blockquote></details>

Done.
___
*[cockroachcloud/statements-page.md, line 43 at r1](https://reviewable.io/reviews/cockroachdb/docs/9917#-MUpLMzmIZ-VN6YhntKR-r1-43:-MXIXZwy3MFx7dYamE_b:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/143258e8c98dfb00e5af400203fc5a365344f8d3/cockroachcloud/statements-page.md#L43)):*
<details><summary><i>Previously, mikeCRL (Mike Lewis) wrote…</i></summary><blockquote>

```suggestion
Diagnostics | Option to activate [diagnostics](#diagnostics) for this fingerprint. If activated, this displays the status of diagnostics collection (`WAITING FOR QUERY`, `READY`, OR `ERROR`). When `READY`, the most recent diagnostics bundle can be downloaded here. Access the full history of diagnostics for the fingerprint in the [**Statement Details**](#statement-details-page) page.
```
</blockquote></details>

Done.

**Comment by yingzhuchin** [comment]
Hi @taroface, can you try pushing again so we trigger a new run in Netlify? We've migrated link checks to Netlify and changed merges to be dependent on Netlify, not TeamCity.

**Comment by taroface** [comment]
@yingzhuchin Unfortunately I am getting the same error in Netlify:

```
11:43:19 AM:   missing reference --- docs/cockroachcloud/statements-page.html --> ../stable/connection-parameters.html#additional-connection-parameters
11:43:19 AM:   missing reference --- docs/cockroachcloud/statements-page.html --> ../stable/transactions.html#transaction-retries
11:43:19 AM:   missing reference --- docs/cockroachcloud/statements-page.html --> ../stable/architecture/transaction-layer.html#transaction-pipelining
11:43:19 AM:   missing reference --- docs/cockroachcloud/statements-page.html --> ../stable/transactions.html#transaction-retries
11:43:19 AM:   missing reference --- docs/cockroachcloud/statements-page.html --> ../stable/transactions.html#transaction-retries
11:43:19 AM:   missing reference --- docs/cockroachcloud/statements-page.html --> ../stable/cluster-settings.html#settings
11:43:19 AM:   missing reference --- docs/cockroachcloud/statements-page.html --> ../stable/show-vars.html#supported-variables
11:43:19 AM:   missing reference --- docs/cockroachcloud/statements-page.html --> ../stable/show-vars.html#supported-variables
11:43:19 AM:   missing reference --- docs/cockroachcloud/statements-page.html --> ../stable/transactions.html#transaction-retries
11:43:19 AM:   missing reference --- docs/cockroachcloud/statements-page.html --> ../stable/query-behavior-troubleshooting.html#visualize-statement-traces-in-jaeger
11:43:19 AM:   missing reference --- docs/cockroachcloud/statements-page.html --> ../stable/explain-analyze.html#debug-option
11:43:19 AM:   missing reference --- docs/cockroachcloud/statements-page.html --> ../stable/query-behavior-troubleshooting.html#query-is-always-slow
11:43:19 AM:   missing reference --- docs/cockroachcloud/statements-page.html --> ../stable/query-behavior-troubleshooting.html#visualize-statement-traces-in-jaeger
11:43:19 AM:   missing reference --- docs/cockroachcloud/statements-page.html --> ../stable/sql-grammar.html#preparable_stmt
11:43:19 AM:   missing reference --- docs/cockroachcloud/statements-page.html --> ../stable/cluster-settings.html#settings
11:43:19 AM:   missing reference --- docs/cockroachcloud/statements-page.html --> ../stable/architecture/sql-layer.html#sql-parser-planner-executor
11:43:19 AM:   missing reference --- docs/cockroachcloud/statements-page.html --> ../stable/make-queries-fast.html#cluster-topology
11:43:19 AM:   missing reference --- docs/cockroachcloud/statements-page.html --> ../stable/transactions.html#transaction-retries
```

Do you think it's possible to make the `stable` links work?

**Comment by yingzhuchin** [comment]
I think it's possible but it will probably require tinkering with `htmltest`, and I'm not sure I have the cycles right now 😞 Let me bring up with @jseldess to see how to prioritize

Meanwhile you can work around it by using the full URL, like "https://www.cockroachlabs.com/docs/stable/connection-parameters.html#additional-connection-parameters" instead of "/stable/connection-parameters.html#additional-connection-parameters". Would that be an acceptable workaround for now?

**Comment by taroface** [comment]
I'll use the version number instead - this is the workaround @lnhsingh has taken in other CC docs. Thanks @yingzhuchin !
